src/hash.cr

```diff
@@ -2154,12 +2155,6 @@ class Hash(K, V)
     def initialize(@hash : UInt32, @key : K, @value : V)
     end
 
-    def self.deleted
```

Comment:
For better or worse this type and method are actually publicly exposed in the API docs. Tho it's fairly obvious it should be an implementation detail...